spec/models/activity_spec.rb

```diff
@@ -5,4 +5,4 @@
       it { should belong_to(:daily_itinerary) }
       it { should have_many(:trip).through(:daily_itinerary) }
    end
-end
```

Comment:
We should add the validations in the tests as well

Reply:
Thanks for the reminder!

Reply:
I've re-pushed